content/docs/user-guide/concepts/data-pipelines.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 'Data Pipelines'
+match: ['data pipeline', 'pipeline', 'pipelines']
+tooltip: >-
+  In DVC, a [data pipeline](/doc/user-guide/concepts/data-pipelines) is a series
+  of data processing stages (for example, console commands that take an input
+  and produce an output). A pipeline may produce intermediate data, and has a
+  final result.
+description: >-
```

Comment:
What about SEO titles?

Reply:
> What about SEO titles?

In our previous discussion we discussed adding the "Concept:" prefix and the decision was to leave it out. That would be the only change I'd make so otherwise the generated titles are OK.

Reply:
> In our previous discussion we discussed adding the "Concept:" prefix and the decision was to leave it out. 

Yes. But I thought we add `titles` everywhere as a "strong suggestion" to Google on what to use as title, since it tends to decide the title on it's own anyway 😆 — you call.

Reply:
Our discussions on metadata have mostly been about the `description`, which does need to be specified because otherwise it defaults to the same copy for every URL.

For the `title` though, if we don't add it in frontmatter it's generated from the label in `sidebar.json`. So we are giving Google a "strong suggestion" for the `title` on each doc URL already. 

My thinking was to only add the title in frontmatter if it needs to be different from the navigation label (to add a prefix, for example). Along the lines of "don't repeat yourself" — but I don't have a problem with being explicit about it either. WDYT?

Reply:
> if we don't add it in frontmatter it's generated from the label in sidebar.json

Ah OK then, I didn't remember that. Thanks